readme.txt

```diff
@@ -40,6 +40,13 @@ tends to be very limited.
 
 == Changelog ==
 
+= 1.0.1 2018-04-13 =
+
+* Added `GitHub Plugin URI` to the plugin header to enable automatic updates in the future. REMOVE THIS IF YOU FORK this extension instead of using it as-is
+* Linked posts now output in the same order as the wp-admin drag-and-drop order instead of alphabetically
```

Comment:
This line (and the next line) in particular doesn't seem to me to be a good fit for the changelog. Aren't they just explaining the extension's purpose? If so, perhaps we should move them to the description area of the plugin (right now, the short and long descriptions are just boilerplate text which ought to be updated anyway).

Reply:
Ignore!